doc/api/async_hooks.md

```diff
@@ -203,13 +203,16 @@ UDPSENDWRAP, UDPWRAP, WRITEWRAP, ZLIB, SSLCONNECTION, PBKDF2REQUEST,
 RANDOMBYTESREQUEST, TLSWRAP
 ```
 
+There is also the `PROMISE` resource type, which is used to track `Promise`
```

Comment:
No just leave it out for now. @trevnorris suggested that maybe we should add all types to the `Providers` list. See https://github.com/nodejs/node/pull/13287#discussion_r119227762